app/models/job.rb

```diff
@@ -14,6 +14,14 @@ class Job < ActiveRecord::Base
   default_scope { by_newest }
 
   def self.visible_for(user)
-    all.where.not(id: user.jobs_hidden.pluck(:id))
+    _not_hidden(user)._not_liked(user)
+  end
+
```

Comment:
I believe that the _ is what you put for private class methods. At this point there isn't a way to make a class method private.